commands/ps/type.js

```diff
@@ -38,7 +38,19 @@ Types: ${cli.color.yellow(formation.map((f) => f.type).join(', '))}`)
 
   let displayFormation = co.wrap(function * () {
     let formation = yield heroku.get(`/apps/${app}/formation`)
+    let shielded = yield heroku.get(`/apps/${app}`)
```

Comment:
nitpick: can this be `const app` and then assign to `const shielded` below? I just find it weird that `shielded` gets reassigned (first an app and then a boolean)..